docroot/themes/humsci/humsci_basic/patterns/accordion/accordion.html.twig

```diff
@@ -0,0 +1,22 @@
+{% if variant == 'open' %}
```

Comment:
Work like this is great for base themes. Would there be any objections if we (SWS) copy some of it into stanford_basic?